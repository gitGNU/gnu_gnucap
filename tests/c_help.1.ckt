#help
.help
.help help
.help fregegiirei
.help help_error_test_with_no_help
.help ?
.help help ?
.help help dasd
.help help subtopic
.help help test
.end
