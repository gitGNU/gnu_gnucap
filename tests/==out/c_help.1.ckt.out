#help 
.help 
      ^ ? no help on topic 
help command
Provides help on a variety of topics
Syntax: help topic
In some cases, help on subtopics is available
Syntax: help topic subtopic
For a list of subtopics: help topic ?

.help fregegiirei 
      ^ ? no help on topic fregegiirei 
.help help_error_test_with_no_help 
      ^ ? no help on topic help_error_test_with_no_help 
.help ? 
      ^ ? no help on topic ? 
  subtopic
  test
.help help dasd 
           ^ ? no help on subtopic dasd 
help subtopic
subtopics give you more detailed information

help test

